libs/portals/admin/ids-admin/src/components/forms/EnvironmentHeader/EnvironmentHeader.tsx

```diff
@@ -35,14 +34,13 @@ export const EnvironmentHeader = ({
   availableEnvironments,
   onChange,
   preHeader,
+  allowPublishing = true,
 }: EnvironmentHeaderProps) => {
   const { formatMessage } = useLocale()
-  const tenant = useRouteLoaderData(tenantLoaderId) as TenantLoaderResult
-  console.log(tenant)
 
-  const options = tenant.availableEnvironments.map((env) =>
+  const options = authAdminEnvironments.map((env) =>
     formatOption(
-      availableEnvironments.includes(env)
+      availableEnvironments.includes(env) || !allowPublishing
```

Comment:
Should skip unavailable environments if `!allowPublishing`, so they don't show up in the list.